Research.md

```diff
@@ -38,6 +38,8 @@ Onchain naming services can be categorized by 4 Types and 3 Data Structures.
 | Astr Domains | .astr | Polkadot Parachain (NPoS, L1) | https://astr.domains/
 | Aurora Name Service | .aurora | Near (PoS 2.0, L2) | https://auroradomains.dev/
 | Avvy Domains | .avax | Avalanche (PoS, L1) | https://avvy.domains/
+| Azero Domains | .azero | Aleph Zero (PoS, L1) | https://azero.domains/
+| Based Names | +name .bitcoin .bit | Bitcoin (PoW, L1) | 
```

Comment:
No link available?

Reply:
This is the site https://based.org/ https://twitter.com/basednames

Unfortunately it's not live yet, so we can either leave it blank until it is, or delete it

Reply:
Let's leave it in and edit it later (in case we forget). Will merge now :)